app/email_sender.py

```diff
@@ -0,0 +1,54 @@
+from settings import Settings
+
+from smtplib import SMTP_SSL as SMTP
+from email.mime.text import MIMEText
+
+from mailjet_rest import Client
+
+
+class EmailSender:
+    """
+    Class to handle the email sending.
+    """
+    def __init__(self):
+        self.settings = Settings()
+        self.email_host = self.settings.EMAIL_HOST
+        self.email_from = self.settings.EMAIL_FROM
+        self.email_from_name = self.settings.EMAIL_FROM_NAME
+        self.email_to = self.settings.EMAIL_TO
+        self.email_username = self.settings.EMAIL_USERNAME
+        self.email_password = self.settings.EMAIL_PASSWORD
```

Comment:
Nie wiem czy ma sens takie rozbijanie. Generalnie to jeśli już chcesz mieć `self.settings` to używaj bezpośrednio wartości gdy potrzebujesz bez przypisywania do np. `self.email_host`.

A tak ogólnie to jeśli settings miałoby być już tą klasą to zrobiłbym to jako argument do `EmailSender`

```
class EmailSender:
    """
    Class to handle the email sending.
    """
    def __init__(self, settings):
        self.settings = settings
```

bo używasz to w kilku miejscach, w `AppRegistrations` też.

A już najlepiej jak już pisałem nie rób z Settings klasy.